angelspetrochem/prototypes/global-override/bobassembly.lua

```diff
@@ -134,6 +134,7 @@ if mods["bobplates"] then
   if angelsmods.trigger.disable_bobs_distilleries then
     angelsmods.functions.add_flag("bob-distillery", "hidden")
     angelsmods.functions.set_next_upgrade("assembling-machine", "bob-distillery", nil)
+    angelsmods.functions.set_next_upgrade("furnace", "bob-distillery", nil)
```

Comment:
Do we need to havet his for assembling machine AND furnace? In case bob changed this to a furnace, shouldn't we delete the assembling machine one?

Reply:
from bobs code directly...
if data.raw.furnace["bob-distillery"] then
  data.raw.furnace["bob-distillery"].next_upgrade = "bob-distillery-2"
else
  data.raw["assembling-machine"]["bob-distillery"].next_upgrade = "bob-distillery-2"
end
the reason I have both in is because of this shenanigan in bobassembly/distillery-update.lua lines 4-8
Without both cases, the next update warning spawns, but with both, it is covered regardless